add time granularity (#3881)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/metricflow-cli.md

</TabItem>
</Tabs>

> ## Time Granularity

Reviewer: our style in the docs is to use sentence case (vs title case) 

```suggestion
## Time granularity
```

> You can optionally specify the time granularity you want your data to be aggregated at by appending double underscore and the unit of granularity you want against your. The granularity options are: day, week, month, quarter year. 

Reviewer 2: ```suggestion
You can optionally specify the time granularity you want your data to be aggregated at by appending double underscore and the unit of granularity you want to the global time dimension `metric_time`. The granularity options to group by are: day, week, month, quarter year. 
```

> You can optionally specify the time granularity you want your data to be aggregated at by appending double underscore and the unit of granularity you want to the global time dimension `metric_time`. The granularity options to group by are: day, week, month, quarter year. 

Reviewer: ```suggestion
Optionally, you can specify the time granularity you want your data to be aggregated at by appending two underscores and the unit of granularity you want to `metric_time`, the global time dimension . You can group the granularity by: `day`, `week`, `month`, `quarter`, and `year`. 
```

Reviewer: i added ticks around the groups because i understood it to mean the "text" to use after the "__". please ignore if i misunderstood

> An example for querying metric data at a monthly grain would look like this

Reviewer: for conciseness

```suggestion
Below is an example for querying metric data at a monthly grain:
```
